AKS/AKSClusterCheck.psm1

```diff
@@ -41,6 +41,17 @@ class AKSClusterCheck: ResourceCheck {
         return $totalNodeCount
     }
 
+    [bool] IsPoolMin3Nodes() {
+        foreach ($nodePool in $this.ClusterObject.agentPoolProfiles) {
+            if ($nodePool.minCount -lt 3) {
+                return $false
+            }
+        }
+        return $true
+    }
+    
+    #create new function that goes through nodepools and checks the minimum if it has atleast 3 nodes. Range od min-max if min is 1 thats not good. If 1 nodepool has min<3 the entire test fails
```

Comment:
As a good practice, comments that are not part of the documentation should be removed from the code.